### PR TITLE
Add nullability annotations

### PR DESCRIPTION
**Pull request by SergioEstevao**

### CODE REVIEW

File: WPXMLRPC/WPXMLRPCDecoder.h [hunk 1]
```diff
@@ -23,8 +23,8 @@
 
 #import <Foundation/Foundation.h>
 
-extern NSString *const WPXMLRPCFaultErrorDomain;
-extern NSString *const WPXMLRPCErrorDomain;
+extern NSString * _Nonnull const WPXMLRPCFaultErrorDomain;
```
Comment (koke) [inline WPXMLRPC/WPXMLRPCDecoder.h]: Did you consider `NS_ASSUME_NONNULL_BEGIN`/`END`?

**Comment by SergioEstevao** [comment]
@koke I followed your suggestions, thanks for the links! Ready for another round!

**Comment by koke** [comment]
I think the common thing with `NS_ASSUME_NONNULL_*` is to wrap the whole `.h` file in it so you never have to use the `nonnull` attribute. You could also use it in `WPXMLRPCEncoder.h`

**Comment by SergioEstevao** [comment]
@koke I update it again, ready for another look.

**Pull request merged**